boardswarm-cli/src/main.rs

```diff
@@ -389,7 +389,7 @@ struct ConfigureArg {
 
 #[derive(Debug, Subcommand)]
 enum Command {
-    /// Configure boardswarm instances
+    /// Configure boardswarm instances access
```

Comment:
```suggestion
    /// Configure client authentication to boardswarm  servers
```
Something like this might be the easiest, trivial, change to remove the confusion a bit ; Without refactoring the subcommand (which seems beyond the scope of this PR) :)